src/controllers/__tests__/tutor.test.js

```diff
@@ -96,8 +96,10 @@ describe('Tutors endpoints /tutors', () => {
         if (err) return done(err);
         expect(res.body).to.be.an('array');
         expect(res.body.length).to.not.equal(0);
-        res.body.forEach((element) => {
-          expect(element.fullName).to.equal(name);
+        res.body.forEach((element, index) => {
```

Comment:
I had the same problem that it never read the fullName virtual property, so I changed something in the tutor model and it works if you want to check to work on the same code.